lib/commands/pairDevice.js

```diff
@@ -15,9 +15,16 @@ const SuitestError = require('../utils/SuitestError');
  * Pair with device
  * @param {Object} instance - main class instance
  * @param {string} deviceId - device to connect to
+ * @param {string} recording - optional, should be 'autostart' | 'manualstart' | 'none'
+ * @param {string} webhookUrl - optional, webhook url
  * @returns {ChainablePromise.<DeviceData>}
  */
-async function pairDevice({webSockets, authContext, logger, pairedDeviceContext}, deviceId) {
+async function pairDevice(
+	{webSockets, authContext, logger, pairedDeviceContext, config},
+	deviceId,
+	recording = 'none',
```

Comment:
I would combine `recording` and `webhookUrl` parameters into object parameters. It will help add additional options in future